<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>about_dialog</class>
 <widget class="QDialog" name="about_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>81</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About this program</string>
  </property>
  <widget class="QLabel" name="lbl_version">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Version 0.3</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_author">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>By Stijn Hinterding</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_author_email">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>s.o.m.hinterding@uu.nl / contact@stijnhinterding.nl</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
